core/workspace_svg.js

```diff
@@ -531,6 +531,15 @@ Blockly.WorkspaceSvg.prototype.getMetricsManager = function() {
 };
 
 /**
+ * Sets the metrics manager for the workspace.
+ * @param {!Blockly.IMetricsManager} metricsManager The metrics manager.
+ */
+Blockly.WorkspaceSvg.prototype.setMetricsManager = function(metricsManager) {
```

Comment:
Do we intend to expose this as a public API? (if not, then it should be annotated as package)

Reply:
done